spec/secret_code_spec.rb

```diff
@@ -44,37 +44,37 @@
         code = SecretCode.new
         code.code = [1, 1, 2, 2]
         guess_s = ['1', '2', '2', '1']
-        expect(code.compare_to_guess(guess_s)).to eq [2, 2]
+        expect(code.compare_to_guess(guess_s)).to_eq [2, 2]
       end
       xit '2 red and 0 white expected' do
         code = SecretCode.new
         code.code = [1, 1, 1, 1]
         guess_s = ['1', '1', '2', '2']
-        expect(code.compare_to_guess(guess_s)).to eq [2, 0]
+        expect(code.compare_to_guess(guess_s)).to_eq [2, 0]
       end
       xit '1 red and 2 white expected' do
         code = SecretCode.new
         code.code = [1, 2, 3, 4]
         guess_s = ['1', '3', '2', '1']
-        expect(code.compare_to_guess(guess_s)).to eq [1, 2]
+        expect(code.compare_to_guess(guess_s)).to_eq [1, 2]
       end
       xit '3 red and 0 white expected' do
         code = SecretCode.new
         code.code = [1, 1, 1, 2]
         guess_s = ['1', '1', '1', '1']
-        expect(code.compare_to_guess(guess_s)).to eq [3, 0]
+        expect(code.compare_to_guess(guess_s)).to_eq [3, 0]
       end
       xit '0 red and 4 white expected' do
         code = SecretCode.new
         code.code = [1, 2, 3, 4]
         guess_s = ['4', '3', '2', '1']
-        expect(code.compare_to_guess(guess_s)).to eq [0, 4]
+        expect(code.compare_to_guess(guess_s)).to_eq [0, 4]
       end
       xit '4 red and 0 white expected' do
```

Comment:
Are these supposed to still be pending? ...Also, I tend to see to eq(THING) more than that way